packages/formspree-react/tsconfig.json

```diff
@@ -3,7 +3,6 @@
   "display": "@formspree/react",
   "extends": ["../../tsconfig.json"],
   "compilerOptions": {
-    "target": "es5",
```

Comment:
Removing this will use the repo-wide target (`es2015`): https://github.com/formspree/formspree-js/blob/bbd3982294dc1b3e1e5aed083403c0818fcb95a5/tsconfig.json#L18, which matches `@formspree/core`.